### DISCUSSION TITLE
Please make Pot Player version

### DISCUSSION THREAD

**Discussion by SeliaLong** [bloc97/Anime4K#229] [Ideas]
Pot player has a 60 fps frame interpolation, its so nice when you watch a movie or anime, can you or someone here make a anime4k version for Pot Player so we can watch anime with high quality and high framerate? 
Please,  this would be so nice

**Comment by Tama47** [comment]
[mpv](https://www.svp-team.com/wiki/SVP:mpv) supports [SVP](https://www.svp-team.com/) if you want interpolation.
